packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -108,8 +136,18 @@ export default function BookmarkFeedLayout({
         )}
       >
         {searchChildren}
-        {shareBookmarksButton('hidden laptop:flex ml-4', 'Share bookmarks')}
-        {shareBookmarksButton('flex laptop:hidden ml-4')}
+        {!isFolderPage && (
+          <ShareBookmarksButton
+            className="ml-4 flex"
+            icon={<ShareIcon secondary={showSharedBookmarks} />}
+            onClick={() => setShowSharedBookmarks(true)}
+          >
+            <span className="hidden laptop:inline">Share bookmarks</span>
```

Comment:
I would like to avoid to add `iconOnly` because many times I found issues on hydration when using `useViewSize` for add/remove props, [before this button didn't have it](https://github.com/dailydotdev/apps/pull/3948/files/0036473ef814dafba417ecbb392b8f8842dece50#diff-2802a541bd3f1481e3cc19c9d15a4a746b9a8a549628bc5a4769c9c6c2ee683aL87) and it is changed only because of a refactor (before was duplicated). Let me know if action is required here ✔️

Reply:
Oh recently fixed it to work on client side here:
https://github.com/dailydotdev/apps/commit/94bd3b9ad751916c19dd53156e4db57b39e5cda9#diff-6c7ea4387e2880c7e85248c8c15f8901aeea14bffd9b2629d1530012f87118f3R48

But up to you, not blocking if it looks ok.